course_content/cartopy_course/solutions/cartopy_exercise_2

```diff
@@ -0,0 +1,24 @@
+
+# Cartopy course, exercise 2
+
+# Use 'set_global' to work out what it was actually doing earlier.
+
```

Comment:
The problem with including this preamble is that it when the user loads in this file, they get a bunch of other stuff, that they don't need as it is clear from the context of the notebook

Reply:
Ok again, I will shorten all these.